tests/publisher_test.py

```diff
@@ -404,3 +404,90 @@ def testUsesUnionOfMetaKeys(self):
     rows = list(reader)
     self.assertEqual(['key1', 'key3'], reader.fieldnames[-2:])
     self.assertEqual(3, len(rows))
+
+
+  class InfluxDBPublisherTestCase(unittest.TestCase):
```

Comment:
This is the line why your tests aren't running.

This whole class is nested inside the CSVPublisherTestCase class. Please unindent and recommit. Thanks!